database/sql/create_data.ddl

```diff
@@ -0,0 +1,6 @@
+DELETE FROM sem_uzivatel WHERE id = 1;
```

Comment:
chcem mazat? jestli jo, tak asi spis vsechny, ne?

Reply:
Podle mě bychom měli mazat jen to, co v insert scriptu vložíme. Kdybychom zapomněli heslo na admina a nechtěli bychom vymazat všechny uživatele.